infra/pom.xml

```diff
@@ -55,6 +55,10 @@
       <groupId>io.quarkus</groupId>
       <artifactId>quarkus-resteasy-qute</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-smallrye-openapi</artifactId>
+    </dependency>
```

Comment:
Is this necessary? I don't see any additional/new usage in the PR

Reply:
removed